### ISSUE TITLE
JOSS paper: State of the field not really described

### ISSUE THREAD

**Issue by fabian-s** [firefly-cpp/NiaARM#66]
re :
>State of the field: Do the authors describe how this software compares to other commonly-used packages?

This seems to be done only very implicitly in the current version of the paper  -- could you summarize e.g. some of the take aways from the Kaushik et al 2021 review of ARM implementations you cite and clarify how your package is different from which other publicly available implementations?

see also openjournals/joss-reviews#4448

**Comment by zStupan** [comment]
@fabian-s

I've updated the statement of need section as requested. I've also added a note to the docs about having to download stopwords and the punkt tokenizer as you've pointed out.
